modules/apps/portal-language/portal-language-override-test/src/testFunctional/tests/LanguageOverrideExportImport.testcase

```diff
@@ -21,6 +21,40 @@ definition {
 		}
 	}
 
+	@description = "This test asserts that Cancel button works as expected in Import Translations page."
+	@priority = "5"
+	test CancelButtonWorks {
+		task ("Given assert the Import Translations page display") {
+			Portlet.gotoPortletOptions(portletOption = "Import Translations");
+
+			ControlMenu.viewHeaderTitle(headerTitle = "Import Translations");
+
+			AssertTextEquals(
+				locator1 = "LanguageOverride#IMPORT_TRANSLATIONS_TITLE",
+				value1 = "Import File");
+
+			AssertTextEquals(
+				locator1 = "LanguageOverride#IMPORT_TRANSLATIONS_DESCRIPTION",
+				value1 = "Support file format .properties.");
+
+			AssertElementPresent(
+				defaultLanguage = "en-us",
+				locator1 = "Select#DEFAULT_LANGUAGE");
+
+			AssertElementPresent(locator1 = "TextInput#FILE");
+
+			AssertElementPresent(locator1 = "Button#SAVE");
```

Comment:
Find the exist macro first.

Reply:
no exist, needs to add and put in LanguagePortlet.macro